easy-connect.cpp

```diff
@@ -223,6 +231,21 @@ NetworkInterface* easy_connect(bool log_messages) {
 #  endif
     connect_success = cellular.connect();
     network_interface = &cellular;
+
+#elif MBED_CONF_APP_NETWORK_INTERFACE == WNC14A2A
+    if (log_messages) {
+        printf("[EasyConnect] Using WNC14A2A\n");
+    }
+#   if MBED_CONF_APP_WNC_DEBUG == true
+        printf("[EasyConnect] With WNC14A2A debug output set to 0x%02X\n",MBED_CONF_APP_WNC_DEBUG_SETTING);
+        wnc = new WNC14A2AInterface(&dbgout);
```

Comment:
Do we have to check for `if (wnc == NULL) ...` and handle the potential allocation failure?

Reply:
You can/should add it, but by default Mbed OS will kill the application when new operator fails.

Reply:
Definiing those then statically at the top would make most sense. It's nicer if it fails in the compilation, rather than runtime.